test/DGmethods_old/soundings/runtests.jl

```diff
@@ -0,0 +1,16 @@
+using DataDeps
+using NCDatasets
+using Test
+
+include("soundings_data.jl")
+
+@testset "Get soundings data" begin
+    data_folder = soundings_data_folder()
+    PYCLES_Z_T_P          = Dataset(joinpath(data_folder, "SOUNDING_PYCLES_Z_T_P.nc"), "r")
```

Comment:
This sounding and all those for Dycoms are now obsolete because we build the dycoms i.c. directly in clima. 
Please, remove 
* SOUNDING_PYCLES_Z_T_P
* sounding_DYCOMS_TEST1

Reply:
The following sounding file should be added to the sounding "data repo" since it is the one being used to initialize the standard squall line benchmark:

`soundings/sounding_gabersek.dat`

https://github.com/climate-machine/CLIMA/blob/8931ed3839d91f20a92560202cd3e9e2a8e4106e/test/DGmethods/compressible_Navier_Stokes/squall-line-model_F3D.jl#L108

Reply:
> This sounding and all those for Dycoms are now obsolete because we build the dycoms i.c. directly in clima.
> Please, remove
> 
> * SOUNDING_PYCLES_Z_T_P
> * sounding_DYCOMS_TEST1

Sounds good, will do.

> The following sounding file should be added to the sounding "data repo" since it is the one being used to initialize the standard squall line benchmark:

Can you point me to where that data file exists? (on a branch/PR)

Reply:
@smarras79 That file, `soundings/sounding_gabersek.dat`, doesn't seem to exist on that branch.

Reply:
Perhaps we can open a new PR for these additional file

Reply:
Ah, nevermind, I found it in #435

Reply:
I'll revive it.